src/tasks/AppleAppBuilder/Xcode.cs

```diff
@@ -211,6 +211,10 @@ public void CreateXcodeProject(string projectName, string cmakeDirectoryPath)
     {
         // bundle everything as resources excluding native files
         var excludes = new List<string> { ".dll.o", ".dll.s", ".dwarf", ".m", ".h", ".a", ".bc", "libmonosgen-2.0.dylib", "libcoreclr.dylib" };
+        if (forceAOT)
```

Comment:
I think if (!preferDylibs) would be better here.

Reply:
make sense